EIPS/eip-5345.md

```diff
@@ -0,0 +1,131 @@
+
+```
+Parameters
+  until: NUMBER - unix timesptamp data 
+
+Returns
+  clientSecret: DATA, 20 Bytes - a secret key for auto-signing requests
+```
+
+#### `wallet_silenSignTransaction`
+This command will create a transaction and send data for sing to the wallet. The wallet will sign data in the background without asking the user. After this, the application can send the signed transaction to the blockchain by using Nethereum or ethers, with sendRawTransaction command.
+
+```
+Parameters
+  clientSecret: DATA, 20 Bytes - secret key obtained from game_requestAutoSign method
+  Object - The transaction object
+    from: DATA, 20 Bytes - The address the transaction is sent from.
+    to: DATA, 20 Bytes - (optional when creating new contract) The address the transaction is directed to.
+    gas: QUANTITY - (optional, default: 90000) Integer of the gas provided for the transaction execution. It will return unused gas.
+    gasPrice: QUANTITY - (optional, default: To-Be-Determined) Integer of the gasPrice used for each paid gas, in Wei.
+    value: QUANTITY - (optional) Integer of the value sent with this transaction, in Wei.
+    data: DATA - The compiled code of a contract OR the hash of the invoked method signature and encoded parameters.
+    nonce: QUANTITY - (optional) Integer of a nonce. This allows to overwrite your own pending transactions that use the same nonce.
+  
+Returns
+  DATA, The signed transaction object.
+```
+
+#### `wallet_silentSendTransaction`
+This command will create a transaction without asking the user and send it to the blockchain.
+
+```
+Parameters
+  clientSecret: DATA, 20 Bytes - secret key obtained from game_requestAutoSign method
+  Object - The transaction object
+    from: DATA, 20 Bytes - The address the transaction is sent from.
+    to: DATA, 20 Bytes - (optional when creating new contract) The address the transaction is directed to.
+    gas: QUANTITY - (optional, default: 90000) Integer of the gas provided for the transaction execution. It will return unused gas.
+    gasPrice: QUANTITY - (optional, default: To-Be-Determined) Integer of the gasPrice used for each paid gas.
+    value: QUANTITY - (optional) Integer of the value sent with this transaction.
+    data: DATA - The compiled code of a contract OR the hash of the invoked method signature and encoded parameters.
+    nonce: QUANTITY - (optional) Integer of a nonce. This allows to overwrite your own pending transactions that use the same nonce.
+
+Returns
+  DATA, 32 Bytes - the transaction hash, or the zero hash if the transaction is not yet available.
+```
+
+### Application and Wallet Communication
+When the wallet application is closed or in the background, it cannot connect to the bridge server via WebSocket. So that we have to trigger the wallet to connect to the bridge and get waiting for requests. For this reason, the Push Notification method should be used. So only the wallets that have push notification support can implement this feature.
```

Comment:
```suggestion

When the wallet application is closed or in the background, it cannot connect to the bridge server via WebSocket. So that we have to trigger the wallet to connect to the bridge and get waiting for requests. For this reason, the Push Notification method should be used. So only the wallets that have push notification support can implement this feature.
```